content/guides/integration-guides/cinder-compatible-SODA-API/_index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Cinder Compatible 
+menuTitle: "SODA API"
```

Comment:
This will be the left menu item. It just says SODA API. 
It should ideally read `With Cinder` similar to `With Kubernetes CSI`
![image](https://user-images.githubusercontent.com/19162717/103399659-12dbaa80-4b68-11eb-966b-53606bcf7b02.png)